src/vs/workbench/contrib/extensions/browser/media/extension.css

```diff
@@ -27,7 +27,7 @@
 .extension-list-item > .icon-container > .icon {
 	width: 42px;
 	height: 42px;
-	padding: 10px 14px 10px 0;
+	padding: 13px 14px 10px 0;
```

Comment:
The icon isn't centered within the list item once the content is correctly centered within the list item (see flexbox suggestion in the other comment).

Would suggest moving this to only apply `padding-right: 14px` and modifying L23 as such:

```
.extension-list-item > .icon-container {
    display: flex;
    align-items: center
}
```

This centers the icon regardless of the list item height and moves the padding rule to the container vs. the icon itself. It also means we can and should remove the padding top here:

https://github.com/microsoft/vscode/blob/bb585dd7fd47ae62479071f673730d887e18a24a/src/vs/workbench/contrib/extensions/browser/media/extensionsViewlet.css#L114